### DISCUSSION TITLE
Can user limit the number of models?

### DISCUSSION THREAD

**Discussion by quinquepeta** [KosinskiLab/AlphaPulldown#369] [Q&A]
Hi, 
Thanks for making this useful program! We just did some test runs with one bait and three candidates. There are alway 5 models predicted. The --num_predictions_per_model=1 is for the complex prediction. Is there a way to limit the number of models? We are thinking of very ambitious runs with thousands of candidates. 5 models per combination may be too much.
Thanks!

**Comment by dingquanyu** [comment]
Hi @quinquepeta ,

Certainly you can limit the total number of models to be used via ```--model_names```flag during run_multimer_job.py. e.g. ```--model_names= model_2_multimer_v3```will only use one model and  ```--model_names=model_2_multimer_v3,model_3_multimer_v3 ``` will use two models. The model_names should be comma-separated in the command line. 

Yours
Dingquan
